tests/e2e/test_chaperone.js

```diff
@@ -126,6 +126,72 @@ describe("Server", () => {
     await closedPromise
   })
 
+  it.only("should return correct appInfo status after app is deactivated", async function() {
```

Comment:
Am I missing something? This test doesn't seem to call appInfo (not directly at least)

Reply:
Test is ready now. :)